geozero/src/geojson/geojson_writer.rs

```diff
@@ -208,6 +208,12 @@ fn write_str_prop<W: Write>(mut out: W, colname: &str, v: &str) -> Result<()> {
     Ok(())
 }
 
+fn write_json_prop<W: Write>(mut out: W, colname: &str, v: &str) -> Result<()> {
+    let colname = colname.replace('\"', "\\\"");
```

Comment:
What's this escaping about? Is this captured in a test somewhere?

Reply:
That's identical to the other `write*` methods
https://github.com/georust/geozero/blob/cdda9a8da2136b2bbda7517561906969296f9081/geozero/src/geojson/geojson_writer.rs#L199
https://github.com/georust/geozero/blob/cdda9a8da2136b2bbda7517561906969296f9081/geozero/src/geojson/geojson_writer.rs#L205

Any embedded `"` in the column name must be escaped with a `\"` or else the JSON will be invalid. I'm not sure if there's an existing test for this, but it would be separate from this change at hand.

Reply:
Ok, I created https://github.com/georust/geozero/issues/209 as a followup task.